packages/app/pages/course/[cid]/today.tsx

```diff
@@ -68,7 +67,7 @@ export default function Today() {
             <Col md={12} xs={24}>
               <Row justify="space-between">
                 <Title>Current Office Hours</Title>
-                {isTA && (
+                {role === Role.TA && (
```

Comment:
fuck i should've looked at this before committing to #81 😭

Reply:
your mom should've looked at this before committing to #81